app/src/org/commcare/CommCareApplication.java

```diff
@@ -540,6 +535,10 @@ public void initializeAppResources(CommCareApp app) {
             FirebaseAnalyticsUtil.reportCorruptAppState();
         }
         app.setAppResourceState(resourceState);
+
+        // This is part of the CommCare app initialization because it needs to be applied during
+        // app initialization, update and when switching the seated app
+        customiseOkHttp();

```

Comment:
@shubham1g5 I thought I had included in the initial PR. Basically, because the Certificate Transparency is a CommCare app level configuration, we need to be able to apply it in case it changes between app updates and when the user switches the seated app.

Reply:
@shubham1g5 true, when CommCare is installed there is not CC app to initialize and the configuration won't applied. I think consolidating configurations of different cycles/types is probably a stretch, but we can continue applying the configuration during app startup, just to cover all the scenarios.

Reply:
[0e53d88](https://github.com/dimagi/commcare-android/pull/2681/commits/0e53d88234bfa72704c8f6fb9e075eedb1896730)